src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/DefaultKfsAccountForAwsWrapperFixture.java

```diff
@@ -0,0 +1,92 @@
+package edu.cornell.kfs.fp.batch.xml.fixture;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Date;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import edu.cornell.kfs.fp.batch.xml.DefaultKfsAccountForAws;
+import edu.cornell.kfs.fp.batch.xml.DefaultKfsAccountForAwsResultWrapper;
+
+public class DefaultKfsAccountForAwsWrapperFixture {
+
+    private static final String EXPECTED_XML_EXAMPLE_FILE_PATH = "src/test/resources/edu/cornell/kfs/fp/batch/xml/default-kfs-account-for-aws-test.xml";
+    private static final String BAD_DATE_XML_EXAMPLE_FILE_PATH = "src/test/resources/edu/cornell/kfs/fp/batch/xml/default-kfs-account-for-aws-bad-date-test.xml";
+    private static final String EMPTY_DATE_XML_EXAMPLE_FILE_PATH = "src/test/resources/edu/cornell/kfs/fp/batch/xml/default-kfs-account-for-aws-empty-date-test.xml";
+
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss");
+
+    public static DefaultKfsAccountForAwsResultWrapper generateExpectedDefaultKfsAccountForAwsResultWrapper() {
+        DefaultKfsAccountForAwsResultWrapper defaultKfsAccountForAwsResultWrapper = new DefaultKfsAccountForAwsResultWrapper();
+
+        ArrayList<DefaultKfsAccountForAws> defaultKfsAccountForAwsList = new ArrayList<>();
+        defaultKfsAccountForAwsList.add(buildDefaultKfsAccountForAws("999913976919", "1719999", "2016-04-26 18:43:03"));

```

Comment:
Sorry for not mentioning the first time, but you could move all these values to private static final variables, and avoid duplicating these strings.